vtex-apps/popover-layout
# Feature/export logic

#### What does this PR do? \*

Split logic and styling in different components. Now the styled-components are used by the blocks and there is some base components in the root of the `/react` folder that are being used by the new `menu` block

#### How to test it? \*

[workspace](https://newmenu--storecomponents.myvtex.com/)
Hover the menu option `More` to see the code being used by the menu, but you can see that all the handles are prefixed by `vtex-menu-2-x-`
